src/actions/web3Actions.ts

```diff
@@ -21,7 +20,6 @@ export function initializeWeb3() {
         type: ActionTypes.WEB3_CONNECTION_REJECTED,
         action: "Can't connect to web3",
       };
-      dispatch(showAlert("Can't connect to web3"));
```

Comment:
so this will be replaced by something else in this PR?

Reply:
I intend to refactor other actions as well the same as the `ARC_STAKE...` actions in this PR. Just wanted to get some feedback before doing so

Reply:
but this is a different kind of alert, we will still want those, for example if a proposal passes after voting on we want a notification for that, and actually im not seeing that here